datastore/db.go

```diff
@@ -421,10 +415,7 @@ func (d *MongoStore) DeleteOne(ctx context.Context, filter bson.M, hardDelete bo
 		return err
 
 	} else {
-		payload := bson.M{
-			"deleted_at":      primitive.NewDateTimeFromTime(time.Now()),
-			"document_status": DeletedDocumentStatus,
-		}
+		payload := bson.M{"deleted_at": primitive.NewDateTimeFromTime(time.Now())}
```

Comment:
@danvixent We didn't apply this to `DeleteMany`

Reply:
fixed